guide/payments/privacy.md

```diff
@@ -0,0 +1,137 @@
+> * He mentions it to someone in a cafe or bar.
+> * Word gets around. A burglar raids your home. Kidnappers capture your children and know exactly how much to demand in ransom.
+
+-->
+
+
+**Do's**
+
+- Make it easy to generate as many addresses as the receiver needs at a time.
+- Before broadcasting a transaction, inform senders if an address is already used.
+
+**Don'ts**
+
+- Make it easy to reuse an address.
+
+### Keep track of who about knows about an address
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/privacy/address-labeling.jpg"
+   retina = "/assets/images/guide/payments/privacy/address-labeling@2x.jpg"
+   alt-text = "A list of addresses with tags. Two are red, two are purple and one is blue"
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+When [receiving bitcoin]({{ "/guide/payments/receive/" | relative_url }}) we can [add additional details to a payment]({{ "/guide/payments/receive/#inputting-additional-payment-details" | relative_url }}). Not only does this help to contextualize payments, but it also enables preventative measures for preserving privacy. Labeling addresses with the counterparty's name involved in each transaction can inform decisions for which [UTXOs]({{ "/guide/glossary/#unspent-transaction-output-utxo" | relative_url }}) are selected in future transaction inputs.
+
+To make this UTXO selection easier, some interfaces make it possible to select/filter UTXOs that cluster under one label.
+
+_Check out the glossary to learn more about [how coin control works]({{ "/guide/glossary/#coin-control" | relative_url }})._
+
+</div>
+
+### Increase anonymity by collaborating with others
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/privacy/coin-join.jpg"
+   retina = "/assets/images/guide/payments/privacy/coin-join@2x.jpg"
+   alt-text = "Example CoinJoin transaction with 3 participants with 1 input each, which get broken up into 5 equal outputs."
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+The "common input ownership" heuristic assumes that in a transaction, it is most likely that all inputs belong to the same owner. [CoinJoins]({{ "/guide/glossary/#coinjoin" | relative_url }}) is a technique where multiple participants collaborate on a transaction to break this heuristic. It becomes more difficult (although not impossible) to determine who the individuals who own the outputs are since all the outputs typically have the same denomination. With CoinJoins, as with any other anonymity network, a large and diverse group of participants provides the best protection.
+
+Users still have to be mindful of how the UTXOs they received from the CoinJoin are spent. For instance, spending them together in a single transaction would unravel the anonymity gains from participating in the CoinJoin.
+
```

Comment:
Can we maybe add some statement on whether this is easy or hard to design or implement, or how many wallets support this?

Reply:
@danielnordh thoughts on this one? not sure how to say if something is hard to design, since once the information is available and there are references it becomes easy. and we're trying to make the resources to make things easier.

Reply:
Added a short statement.